packages/mdc-textfield/foundation.js

```diff
@@ -83,6 +100,16 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.showHelptext_();
   }
 
+  autoCompleteFocus_(keyInteraction) {
```

Comment:
I feel like we don't need to pass in a `keyInteraction` argument here, since we already have `keyInteractionOccurred` as an instance property. So it can just be

```js
if (!this.keyInteractionOccurred) {
  this.activateFocus_();
}